core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -21,6 +21,13 @@
   <div class="protractor-test-conversation-input"
        angular-html-bind="activeCard.interactionHtml">
   </div>
+  <div ng-if="currentInteractionHints.length" style="text-align: right; padding: 8px">
+    <hint-button on-click-hint-button="processHintRequest()"
+                 is-disabled="!isHintAvailable()"
```

Comment:
Do you actually need to pass in is-disabled() here, if you're already passing in current-hint-is-available below which tracks the same thing?